testing/python/language/test_tilelang_language_warp_vote.py

```
@@ -0,0 +1,327 @@
+# ---------------------------------------------------------------------------
+# any_sync
+# ---------------------------------------------------------------------------
+
+
+@tilelang.jit
+def kernel_any_sync():
+    """Lane 0 writes 1 to A; all lanes use any_sync to see if any lane wrote."""
+
+    @T.prim_func
+    def main(
+        A: T.Tensor((1,), "int32"),
```

Ruff (0.15.1)

[warning] Unused function argument: `A`

(ARG001)